content/testing/functional.md

```diff
@@ -1,37 +1,31 @@
-# Functional Testing
+Functional Testing
+==================
 
 Check if a site function properly.
 
-## Two main categories:
-1. cross-browser (css/js front-end perfection)
-2. single-browser (raw functioanlity tests)
+Two main categories:
+--------------------
 
+1.	cross-browser (css/js front-end perfection)
+2.	single-browser (raw functioanlity tests)
 
-## Tools
+Tools
+-----
```

Comment:
Its seems you are using some markdown tool?

Please use gitbook editor or some other editor which doesn't do auto-formatting like this.